string model_name
PoseStamped[] next_poses
Twist model_twist